toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/GraphicsOverlayCollection.kt

```diff
@@ -0,0 +1,105 @@
+/*
+ *  Copyright 2023 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.arcgismaps.toolkit.geocompose
+
+import com.arcgismaps.mapping.view.GraphicsOverlay
+import kotlinx.coroutines.channels.BufferOverflow
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.SharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
+
+/**
+ * A collection class to encapsulate the [GraphicsOverlay] list used by the [com.arcgismaps.toolkit.geocompose.MapView]
+ *
+ * @since 200.3.0
+ */
+public class GraphicsOverlayCollection : Iterable<GraphicsOverlay> {
```

Comment:
You can annotate this class with `@Stable`:
```suggestion
@Stable
public class GraphicsOverlayCollection : Iterable<GraphicsOverlay> {
```